plugins/elevenlabs/src/tts.ts

```diff
@@ -278,6 +278,12 @@ export class SynthesizeStream extends tts.SynthesizeStream {
                 lastFrame = frame;
               }
               sendLastFrame(segmentId, true);
+              this.queue.put(SynthesizeStream.END_OF_STREAM);
+
+              if (segmentId === requestId) {
+                ws.close();
```

Comment:
what happens to the ws connection if this condition is not true?

Reply:
then it will keep going. END_OF_STREAM is just the end of the current utterance. this close is just making sure on our end that the connection closes when it needs to, even though it should be closed on the provider's end

Reply:
got it, thanks!